src/tools/tidy/src/deps.rs

```diff
@@ -268,6 +268,14 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "winapi-i686-pc-windows-gnu",
     "winapi-util",
     "winapi-x86_64-pc-windows-gnu",
+    "windows-targets",
```

Comment:
So this seems to be required now because in the version change, the way the `windows-targets` crate was included has changed. [before](https://docs.rs/crate/windows/0.46.0/source/Cargo.toml):

```toml
[target."cfg(not(windows_raw_dylib))".dependencies.windows-targets]
version = "0.42.2"
```

[after](https://docs.rs/crate/windows/0.48.0/source/Cargo.toml):

```toml
[dependencies.windows-targets]
version = "0.48.0"
```

Reply:
I guess cargo metadata just considered the non raw-dylib case before.